### PR TITLE
Implement Vector Class - int indexOf(Object elem, int index)

### LINKED ISSUE #220: Implement Vector Class - int indexOf(Object elem, int index)

### PR DESCRIPTION
**Pull request by vatsalkesarwani12**
Fixes #220 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- Java
- 

#### Files Added:
- int indexOf(Object elem, int index)
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

Keep Contributing. Keep Learning!

### CODE REVIEW

File: Collection Classes/Vector Class/int indexOf(Object elem, int index)/int indexOf(Object elem, int index).java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    {
+	Scanner sc=new Scanner ( System.in );
+	    
+	//n is the size of vector
+	int n = sc.nextInt();
+	    
+        // Declare an empty vector
+         Vector<String> vc = new Vector<>(); 
+         
+         Vector<String> vc1 = new Vector<>();
```
Comment (jyotijauhari) [inline Collection Classes/Vector Class/int indexOf(Object elem, int index)/int indexOf(Object elem, int index).java]: need of vc1? @plazzy99

**Review by jyotijauhari — CHANGES_REQUESTED** [review]
need of vc1?

File: Collection Classes/Vector Class/int indexOf(Object elem, int index)/int indexOf(Object elem, int index).java (hunk 1 above)
Comment (vatsalkesarwani12) [reply Collection Classes/Vector Class/int indexOf(Object elem, int index)/int indexOf(Object elem, int index).java]: ohh i just used the previous code as template of this one so by mistake

**Review by vatsalkesarwani12 — COMMENTED** [review]
@JyotiJauhari changes done

**Review by jyotijauhari — APPROVED** [review]

**Pull request merged**